pkg/kubelet/cm/dra/manager.go

```diff
@@ -99,7 +99,7 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 				// Check if pod is in the ReservedFor for the claim
 				if !resourceclaim.IsReservedForPod(pod, resourceClaim) {
 					return fmt.Errorf("pod %s(%s) is not allowed to use resource claim %s(%s)",
-						pod.Name, pod.UID, podResourceClaim.Name, resourceClaim.UID)
+						pod.Name, pod.UID, claimName, resourceClaim.UID)
```

Comment:
This looks like an unrelated fix. It makes sense, though.

Reply:
It’s somewhat unrelated. It does bring this function in line with what is done in the additions to the unprepared call, which is why it felt relevant.